leveldb/session_record_test.go

```diff
@@ -46,11 +46,21 @@ func TestSessionRecord_EncodeDecode(t *testing.T) {
 
 	for ; i < 4; i++ {
 		test()
-		v.addTable(3, big+300+i, big+400+i,
-			makeInternalKey(nil, []byte("foo"), uint64(big+500+1), keyTypeVal),
-			makeInternalKey(nil, []byte("zoo"), uint64(big+600+1), keyTypeDel))
+		ik1, err1 := makeInternalKey(nil, []byte("foo"), uint64(big+500+1), keyTypeVal)
+		if err1 != nil {
+			panic(err1) // Test code - should not fail with valid parameters
```

Comment:
Similar to the previous comment on the tests -- since we do already have access to `t *testing.T` here, I do think using an assert or require instead of a panic here would be cleaner and give us a better error message when the test fails. Same goes for `panic(err2)` and `panic(err3)` below.